api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -76,6 +76,13 @@ pub enum GetOpenDepositChannelsQuery {
 	Mine,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct VaultAddresses {
+	ethereum: String,
+	arbitrum: String,
+	bitcoin: String,
```

Comment:
For bitcoin there should be multiple addresses: each broker can have one private channel and each private channel needs two addresses: one the current and one for the previous epoch's key.